watertap/examples/flowsheets/mvc/mvc_single_stage.py

```diff
@@ -0,0 +1,745 @@
+    iscale.set_scaling_factor(m.fs.hx_brine.area, 1e-1)
+    iscale.constraint_scaling_transform(m.fs.hx_brine.cold.pressure_drop, 1e-5)
+    iscale.constraint_scaling_transform(m.fs.hx_brine.hot.pressure_drop, 1e-5)
+
+    # evaporator
+    iscale.set_scaling_factor(m.fs.evaporator.area, 1e-3)
+    iscale.set_scaling_factor(m.fs.evaporator.U, 1e-3)
+    iscale.set_scaling_factor(m.fs.evaporator.delta_temperature_in, 1e-1)
+    iscale.set_scaling_factor(m.fs.evaporator.delta_temperature_out, 1e-1)
+    iscale.set_scaling_factor(m.fs.evaporator.lmtd, 1e-1)
+
+    # compressor
+    iscale.set_scaling_factor(m.fs.compressor.control_volume.work, 1e-6)
+
+    # condenser
+    iscale.set_scaling_factor(m.fs.condenser.control_volume.heat, 1e-6)
+
+    # calculate and propagate scaling factors
+    iscale.calculate_scaling_factors(m)
+
+    return m
+
+
+def add_Q_ext(m, time_point=None):
+    # Allows additional heat to be added to evaporator so that an initial feasible solution can be found as a starting
+    # guess for optimization in case physically infeasible simulation is proposed
+
+    if time_point is None:
+        time_point = m.fs.config.time
+    m.fs.Q_ext = Var(time_point, initialize=0, units=pyunits.J / pyunits.s)
+    m.fs.Q_ext[0].setlb(0)
+    m.fs.evaporator.eq_energy_balance.deactivate()
+    m.fs.evaporator.eq_energy_balance_with_additional_Q = Constraint(
+        expr=m.fs.evaporator.heat_transfer
+        + m.fs.Q_ext[0]
+        + m.fs.evaporator.properties_feed[0].enth_flow
+        == m.fs.evaporator.properties_brine[0].enth_flow
+        + m.fs.evaporator.properties_vapor[0].enth_flow_phase["Vap"]
+    )
+    iscale.set_scaling_factor(m.fs.Q_ext, 1e-6)
+
+
+def add_costing(m):
+    m.fs.costing = WaterTAPCosting()
+    m.fs.pump_feed.costing = UnitModelCostingBlock(flowsheet_costing_block=m.fs.costing)
+    m.fs.pump_distillate.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing
+    )
+    m.fs.pump_brine.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing
+    )
+    m.fs.hx_distillate.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing
+    )
+    m.fs.hx_brine.costing = UnitModelCostingBlock(flowsheet_costing_block=m.fs.costing)
+    m.fs.mixer_feed.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing
+    )
+    m.fs.evaporator.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing
+    )
+    m.fs.compressor.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing
+    )
+
+    m.fs.costing.cost_process()
+    m.fs.costing.add_annual_water_production(m.fs.distillate.properties[0].flow_vol)
+    m.fs.costing.add_LCOW(m.fs.distillate.properties[0].flow_vol)
+    m.fs.costing.add_specific_energy_consumption(m.fs.distillate.properties[0].flow_vol)
+
+
+def add_pressure_drop_to_hx(hx_blk, time_point):
```

Comment:
Interesting- I wonder if IDAES should include pressure drop in HX units.

Reply:
@adam-a-a IDAES does support pressure change in HX units - each side has a `has_pressure_change` option.

Reply:
Ah, true. I had quickly jumped to the 0D HX and searched for "has_pressure" or "deltaP" and nothing turned up, but the control volumes are set up in the `heater` unit, which is where you'll find the `has_pressure_change` option and also see that `deltaP` is accounted for in the model. 

So then- is there a good reason to have this `add_pressure_drop_to_hx` that I'm missing?